### PR TITLE
Release Notes for 1.9.4

### PR DESCRIPTION
**Pull request by shamsher31**

### CODE REVIEW

File: content/en/news/releases/1.9.x/announcing-1.9.4/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - /news/announcing-1.9.4
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.9.3 and Istio 1.9.4
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** Istio operator prunes the resource that doesn't belong to specific Istio operator CR. ([Issue #30833](https://github.com/istio/istio/issues/30833))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.4/index.md:18]: **Fixed** an issue where the Istio operator prunes ...

File: content/en/news/releases/1.9.x/announcing-1.9.4/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- **Fixed** ensure lease duration is always larger than the user configured `RENEW_DEADLINE` for Istio operator manager. ([Issue #27509](https://github.com/istio/istio/issues/27509))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.4/index.md:20]: **Fixed** an issue ensuring lease duration is always **greater** ....

File: content/en/news/releases/1.9.x/announcing-1.9.4/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- **Fixed** an issue where certificate provisioned by sidecar proxy cannot be used by Prometheus. ([Issue #29919](https://github.com/istio/istio/issues/29919))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.4/index.md:22]: where **a** certificate provisioned

File: content/en/news/releases/1.9.x/announcing-1.9.4/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- **Fixed** an issue that creates an IOP under `istio-system` when installing Istio in another namespace. ([Issue #31517](https://github.com/istio/istio/issues/31517))
+
+- **Fixed** When using PeerAuthentication to turn off mTLS while using multi-network, non-mTLS endpoints will be removed from the cross-network load-balancing endpoints to prevent 500 errors. ([Issue #28798](https://github.com/istio/istio/issues/28798))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.4/index.md:26]: **Fixed** an issue when using ...

File: content/en/news/releases/1.9.x/announcing-1.9.4/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- **Improved** the `istioctl x workload` command to configure VMs to disable inbound `iptables` capture for admin ports, matching behavior of Kubernetes Pods. ([Issue #29412](https://github.com/istio/istio/issues/29412))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.4/index.md:28]: matching **the** behavior

**Review by ericvn — COMMENTED** [review]
Some minor grammar changes.

**Comment by shamsher31** [comment]
@ericvn I haved updated the suggestion and added a new commit https://github.com/shamsher31/istio.io/commit/d1bc4b4c33b17d370354d590d1bcb2a30fd7e0b8

But the PR is not picking up the updates. Not sure why.

**Comment by shamsher31** [comment]
I will close this and create a new PR.

**Comment by shamsher31** [comment]
Not sure what went wrong in this PR but it's not picking up the new commits. Created new PR and added suggestions in that PR https://github.com/istio/istio.io/pull/9593

**Pull request closed without merging**